# Removes check on crossbuild-dependent variable set in toplevel configure.ac
diff --git a/extras/Makefile.am b/extras/Makefile.am
index d3a800f49a..324575ad4d 100644
--- a/extras/Makefile.am
+++ b/extras/Makefile.am
@@ -20,9 +20,7 @@
 # along with GnuCOBOL.  If not, see <https://www.gnu.org/licenses/>.
 
 extrasdir = @COB_LIBRARY_PATH@
-if COB_MAKE_RUN_BINARIES
 extras_DATA = CBL_OC_DUMP.$(COB_MODULE_EXT)
-endif
 COBC = cobc$(EXEEXT)
 
 EXTRA_DIST = CBL_OC_DUMP.cob
